### PR TITLE
export object page: allow exporting list of objs

### PR DESCRIPTION
**Pull request by larshp**
allow exporting a list of object names, instead of single value,

image

### CODE REVIEW

File: src/ui/zcl_abapgit_gui_page_ex_object.clas.abap [hunk 1]
```diff
@@ -101,11 +111,13 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_EX_OBJECT IMPLEMENTATION.
       iv_name        = c_id-object_type
       iv_required    = abap_true
       iv_upper_case  = abap_true
-      iv_side_action = c_event-choose_object_type
-    )->text(
+      iv_side_action = c_event-choose_object_type ).
+
+    ro_form->textarea(
       iv_label       = 'Object Name'
```
Comment (mbtools) [inline src/ui/zcl_abapgit_gui_page_ex_object.clas.abap:117]: Names

**Review by mbtools — COMMENTED** [review]
Nice. Let's change the page title and menu item to plural "Export Objects"

**Review by mbtools — APPROVED** [review]
👍

**Comment by mbtools** [comment]
Prompts for download folder once per object but works fine.

**Review by mbtools — APPROVED** [review]

**Comment by larshp** [comment]
folder per object: yea, I know, but much faster than before to export 5 objects

**Pull request merged**